src/main/java/oss/fosslight/service/impl/ApiProjectServiceImpl.java

```diff
@@ -512,6 +518,49 @@ public Map<String, Object> selectVerificationCheck(String prjId){
 	public boolean updatePackageFile(Map<String, Object> paramMap) {
 		return apiProjectMapper.updatePackageFile(paramMap) > 0;
 	}
+
+	@Override
+	public Map<String, Object> getBomExportJson(String prjId) {
+		// Json Format: Yaml Format + Vulnerability
+
+		// Get Yaml Format
+		String type = CoConstDef.CD_DTL_COMPONENT_ID_BOM;
+		Project project = new Project();
+		project.setPrjId(prjId);
+
+		String dataStr = toJson(project);
+		Type projectType = new TypeToken<Project>(){}.getType();
+		Project projectBean = (Project) fromJson(dataStr, projectType);
+
+		ProjectIdentification _param = new ProjectIdentification();
+		_param.setReferenceDiv(type);
+		_param.setReferenceId(projectBean.getPrjId());
+		_param.setMerge(CoConstDef.FLAG_NO);
+
+		Map<String, Object> map = projectService.getIdentificationGridList(_param, true);
+		List<ProjectIdentification> list = (List<ProjectIdentification>) map.get("rows");
+
+		LinkedHashMap<String, List<Map<String, Object>>> resultYamlFormat = YamlUtil.checkYamlFormat(projectService.setMergeGridData(list), type);
+
+		List<Map<String, Object>> bomList = getBomList(prjId);

```

Comment:
The value of the local variable `bomList` is not used. Is there a reason why you declared it?

Reply:
@jongwooo Thank you!! I fixed it 😃